pkg/kics/service.go

```diff
@@ -61,18 +61,18 @@ type Service struct {
 }
 
 // PrepareSources will prepare the sources to be scanned
-func (s *Service) PrepareSources(ctx context.Context, scanID string, wg *sync.WaitGroup, errCh chan<- error) {
+func (s *Service) PrepareSources(ctx context.Context, scanID string, wg *sync.WaitGroup, errCh chan<- error, resolveReferences bool) {
```

Comment:
Nitpick: When adding variables to methods or functions i usually add at the end, however, in this case it seems a bit strange to add the variable at the end, as the last 2 seem more general and usually take up the last spots, I would advise changing the signature so that the added variable comes after the scanId for example